src/main/java/at/hannibal2/skyhanni/config/features/misc/MiscConfig.java

```diff
@@ -184,6 +184,12 @@ public class MiscConfig {
     @FeatureToggle
     public boolean accountUpgradeReminder = true;
 
+    @Expose
+    @ConfigOption(name = "Fame Cost", desc = "Show cost equivalent for your fame in the lore of your fame rank")
```

Comment:
I feel like this should clarify using gems, some people might see this as "useful for knowing how many booster cookies to buy".